default.nix

```diff
@@ -65,21 +65,51 @@ with holonix.pkgs;
 
    ++ ([(
     holonix.pkgs.writeShellScriptBin "hc-install" ''
-     cargo install --path crates/holochain
-     echo 'holochain installed!'
-     echo
+     hc-install-holochain
+     hc-install-dna-util
 
      hc-doctor
     ''
    )])
 
    ++ ([(
     holonix.pkgs.writeShellScriptBin "hc-uninstall" ''
+     hc-uninstall-holochain
+     hc-uninstall-dna-util
+
+     hc-doctor
+    ''
+   )])
+
+   ++ ([(
+    holonix.pkgs.writeShellScriptBin "hc-install-holochain" ''
+     cargo install --path crates/holochain
+     echo 'holochain installed!'
+     echo
+    ''
+   )])
+
+   ++ ([(
+    holonix.pkgs.writeShellScriptBin "hc-uninstall-holochain" ''
      rm $CARGO_INSTALL_ROOT/bin/holochain
      echo 'holochain uninstalled!'
      echo
+    ''
+   )])
 
-     hc-doctor
+   ++ ([(
+    holonix.pkgs.writeShellScriptBin "hc-install-dna-util" ''
+     cargo install --path crates/dna_util
+     echo 'dna util installed!'
+     echo
+    ''
+   )])
+
+   ++ ([(
+    holonix.pkgs.writeShellScriptBin "hc-uninstall-dna-util" ''
+     rm $CARGO_INSTALL_ROOT/bin/dna-util
```

Comment:
Why not `cargo uninstall` if nothing else just for symmetry with the install command?

Same with `hc-uninstall-holochain` above.